.github/workflows/e2e.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 env:
   REGISTRY: 'ghcr.io/dell/csi-baremetal'
-  go_version: '1.19.2'
+  go_version: '1.21.3'
   golangci_version: '1.48.0'
```

Comment:
I see other golangci-lint version changed to 1.55.2,  should we also change this together?

Reply:
Done